### PR TITLE
add user guide to atomic registry distro

### PR DESCRIPTION
**Pull request by aweiteka**
This creates a User Guide for atomic-registry distro for topics lost in #1890 

cc @adellape

### CODE REVIEW

File: dev_guide/authentication.adoc [hunk 1]
```diff
@@ -124,7 +129,7 @@ link:../cli_reference/manage_cli_profiles.html[manage multiple CLI profiles].
 [NOTE]
 ====
 If you have access to administrator credentials but are no longer logged in as
-the link:/architecture/core_concepts/projects_and_users.html#users[default
+the link:../architecture/core_concepts/projects_and_users.html#users[default
```
Comment (aweiteka) [inline dev_guide/authentication.adoc]: FYI, bugfix

File: dev_guide/managing_images.adoc [hunk 2]
```diff
@@ -455,6 +456,7 @@ mountable inside of a pod. You can do this by running:
 ----
 $ oc secrets add serviceaccount/builder secrets/<pull_secret_name>
 ----
+endif::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
```
Comment (adellape) [inline dev_guide/managing_images.adoc]: @aweiteka You are currently leaving out the "Accessing the Internal Registry" section. Is that intended? There is an anchor reference to it later in the "Creating an Image Stream by Manually Pushing an Image" section.

**Comment by adellape** [comment]
@aweiteka 1 comment inline. Also

- Saw about 5 or so instances of "OpenShift" in the auth topic that could become `{product-title}`.
- There are a couple references in the managing_images topic to two dev_guide topics that don't appear in the AR build, for `secrets` and `service_accounts`.

File: dev_guide/managing_images.adoc (hunk 2 above)
Comment (aweiteka) [inline dev_guide/managing_images.adoc]: Good catch. Thanks.

**Comment by aweiteka** [comment]
@adellape updated and squashed.

**Comment by adellape** [comment]
@aweiteka Changes look good. Just saw this though that comes out weird for AR:

```
[IMPORTANT]
====
Avoid tagging internally managed images (those built and pushed locally to an
internal registry). There is a
ifdef::openshift-origin,openshift-enterprise,atomic-registry[]
link:../install_config/install/docker_registry.html#registry-known-issues[known
issue]
endif::[]
ifdef::openshift-online,openshift-dedicated[]
preventing the registry client from pulling from such a tag.
endif::[]
====
```

Think it's just a bad ifdef from me earlier.

**Comment by adellape** [comment]
@aweiteka LMK when that Important box is fixed up and I'll merge this.

File: dev_guide/managing_images.adoc [hunk 3]
```diff
@@ -105,13 +105,11 @@ Image Metadata] for more details.
...
+internal registry).
 ifdef::openshift-origin,openshift-enterprise,atomic-registry[]
+There is a
 link:../install_config/install/docker_registry.html#registry-known-issues[known
-issue]
-endif::[]
-ifdef::openshift-online,openshift-dedicated[]
-preventing the registry client from pulling from such a tag.
+issue] preventing the registry client from pulling from such a tag.
 endif::[]
```
Comment (adellape) [inline dev_guide/managing_images.adoc]: @aweiteka Suggest this for the Important admonition:

```
[IMPORTANT]
====
Avoid tagging OpenShift-managed images (i.e., those built using an OpenShift instance and pushed to its internal registry). There is a
ifdef::openshift-origin,openshift-enterprise[]
link:../install_config/install/docker_registry.html#registry-known-issues[known
issue]
endif::[]
ifdef::openshift-online,openshift-dedicated,atomic-registry[]
https://docs.openshift.com/enterprise/latest/release_notes/ose_3_2_release_notes.html#ose-32-known-issues[known issue]
endif::[]
that prevents the registry client from pulling from such a tag.
====
```

**Comment by aweiteka** [comment]
@adellape updated, squashed

File: dev_guide/index.adoc [hunk 4]
```diff
@@ -5,13 +5,27 @@
...
+ifdef::atomic-registry[]
+This guide helps image developers set up and configure a workstation to
+develop images working with the {product-title} environment using the web console
+and command-line interface (CLI). This guide provides detailed instructions and
+examples to help image developers:
+
+. link:authentication.html[Configure and utilize the CLI]
+. link:managing_images.html[Managing images]
+endif::atomic-registry[]
+>>>>>>> add user guide to atomic registry distro
```
Comment (adellape) [inline dev_guide/index.adoc]: @aweiteka Conflict remnant here.

File: dev_guide/managing_images.adoc [hunk 5]
```diff
@@ -104,15 +104,15 @@ Image Metadata] for more details.
...
-ifdef::openshift-origin,openshift-enterprise,atomic-registry[]
+Avoid tagging OpenShift-managed images (i.e., those built using an OpenShift instance and pushed to its internal registry). There is a
+ifdef::openshift-origin,openshift-enterprise[]
 link:../install_config/install/docker_registry.html#registry-known-issues[known
 issue]
 endif::[]
-ifdef::openshift-online,openshift-dedicated[]
-preventing the registry client from pulling from such a tag.
+ifdef::openshift-online,openshift-dedicated,atomic-registry[]
+https://docs.openshift.com/enterprise/latest/release_notes/ose_3_2_release_notes.html#ose-32-known-issues[known issue]
```
Comment (adellape) [inline dev_guide/managing_images.adoc]: s/latest/3.2/

For when "latest" is incremented to the next version.

**Comment by adellape** [comment]
@aweiteka Two minor things inline.

**Comment by aweiteka** [comment]
@adellape updated, thx.

**Comment by adellape** [comment]
:+1: 

No revhistory needed.

**Pull request merged**